keylime/tenant.py

```diff
@@ -662,7 +662,14 @@ def do_cv(self):
 
         if response.status_code == 409:
             # this is a conflict, need to update or delete it
-            logger.error("Agent %s already existed at CV. Please use delete or update.", self.agent_uuid)
+            message = response.json()["status"]
```

Comment:
If you make the text of the HTTP error message more user friendly then you can just re-use it here (for both of these cases) instead of trying to parse it and give some other take on it.